src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -23,7 +23,7 @@ public class LocalLinuxTransport : StreamTransport
         private string _gdbStdOutName;
         private FileSystemWatcher _fifoWatcher;
 
-        private void MakeGdbFifo(string path)
+        public static void MakeGdbFifo(string path, Logger logger)
```

Comment:
This was part of the work for another fix related to Mac debugging. I'll remove it from this checkin.